src/utils.js

```diff
@@ -103,49 +103,6 @@ export function combineRacedRequests (key, fn) {
   return pending[key].then(handleResponse)
 }
 
-/**
- * Reducer function for filtering out duplicate records
- * by a key provided. Returns a function that has a accumulator and
- * current record per Array.reduce.
- *
- * @method uniqueByReducer
- * @param {Array} key
- * @return {Function}
- */
-export function uniqueByReducer (key) {
-  return function (accumulator, current) {
-    return accumulator.some(item => item[key] === current[key])
-      ? accumulator
-      : [...accumulator, current]
-  }
-}
-
-/**
- * Returns objects unique by key provided
- *
- * @method uniqueBy
- * @param {Array} array
- * @param {String} key
- * @return {Array}
- */
-export function uniqueBy (array, key) {
-  return array.reduce(uniqueByReducer(key), [])
-}
-
-export function stringifyIds (object) {
-  Object.keys(object).forEach(key => {
-    const property = object[key]
-
-    if (typeof property === 'object') {
-      if (property.id) {
-        property.id = String(property.id)
-      }
-
-      stringifyIds(property)
-    }
-  })
-}
```

Comment:
It doesn't look like we use this stringifyIds anywhere - we have another one on the Model